### PR TITLE
Java. Add FacebookPublisherService, configs, edit pom.xml

### PR DESCRIPTION
**Pull request by derkenbaev**
Add FacebookPublisherService
Add Facebook Configurations
Add dependencies to pom.xml
Service can post to the test page of the test user, but it's not integrated with sharing microservice yet. It will be done later. Moreover, it only works with test user now. Posting to real user's page will be completed later too.

### CODE REVIEW

File: share/src/main/java/io/zensoft/share/service/FacebookPublisherService.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+        userAccessToken = getTestUserAccessToken();
+        facebookUser = new FacebookTemplate(userAccessToken, properties.getProperty("appNamespace"), properties.getProperty("appId"));
+
+        pageAccessToken = getPageAccessToken();
+        facebookPage = new FacebookTemplate(pageAccessToken, properties.getProperty("appNamespace"), properties.getProperty("appId"));
+    }
+
+
+    public static void main(String[] args) throws Exception {
```
Comment (PlusRT) [inline share/src/main/java/io/zensoft/share/service/FacebookPublisherService.java]: i think we have already main method)

File: share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+appId=1482536371851716
+appSecret=xxx
+pageId=179314156095163
+userId=103714360544349
```
Comment (NickRoZensoft) [inline share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig]: this is security sensitive info, please add `config.example` to git.  `config` file should be in gitignore

File: share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig [hunk 3]
```diff
@@ -0,0 +1,6 @@
+appId=1482536371851716
```
Comment (PlusRT) [inline share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig]: May be we should keep this file in resources folder? how do you think @derkenbaev

File: share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig (hunk 2 above)
Comment (kb1prb13) [reply share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig]: Ok, will be corrected

Comment (derkenbaev) [reply share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig]: Is ID also a security sensitive info? It is easy to find facebook ID of any page or user

File: share/src/main/java/io/zensoft/share/service/FacebookPublisherService.java (hunk 1 above)
Comment (derkenbaev) [reply share/src/main/java/io/zensoft/share/service/FacebookPublisherService.java]: Yes, I add it to work with my class. I'll remove it now. Thank you for your comment!

**Pull request merged**